src/MOI/MOI_wrapper.jl

```diff
@@ -84,7 +85,8 @@ mutable struct VariableInfo
             "",
             "",
             NaN,
-            0
+            0,
+            NaN
```

Comment:
there is something off about the number of arguments that is no matching with the number of attributes

Reply:
The attributes previous_lower_bound and previous_upper_bound weren't initialized in the arguments of the constructor. But this problem is still on the master branch

Reply:
got it! good that you fixed.